.changeset/tired-places-cough.md

```diff
@@ -0,0 +1,27 @@
+---
+"fluid-framework": minor
+"@fluidframework/tree": minor
+---
+---
+"section": feature
+---
+
+Provide access to tree content and stored schema without requiring a compatible view schema
```

Comment:
Is this still accurate? My main nit about the existing version if that a compatible view schema seems to still be required to access the data in particular, but now we can generate it at runtime.
```suggestion
Provide alpha APIs for accessing tree content and stored schema when the view schema isn't known at compile time
```

Reply:
This PR exposes ITreeAlpha, which you can get without a view schema, that has:
```typescript
 exportSimpleSchema(): SimpleTreeSchema;
exportVerbose(): VerboseTree | undefined;
```

Which allow access to the schema and content respectively.

It is important to note that these can be useful if even if you have a view schema but it isn't compatible with the document. Thus I think my original phrasing is more relevant than the proposed change here.

Reply:
I have applied part of this suggestion